### ISSUE TITLE
run_bepinex.sh line endings aren't unix's

### ISSUE THREAD

**Issue by siddhartasr10** [BepInEx/BepInEx#709]
### What happened?

Line terminators in the run_bepinex.sh script aren't unix ones so executing the script without changing them gives an error.

This is very easy to fix, but if you don't know what is happening or are executing it with sh instead of directly as binary the error message will be different.

### Steps to reproduce

1. Download the Unix release.
2. Run the `run_bepinex.sh` script for the first time.
3. Get an error: 
```
. ./run_bepinex.sh   
./run_bepinex.sh:8: command not found: ^M
./run_bepinex.sh:11: command not found: ^M
./run_bepinex.sh:16: command not found: ^M
./run_bepinex.sh:18: command not found: ^M
./run_bepinex.sh:21: command not found: ^M
./run_bepinex.sh:24: command not found: ^M
./run_bepinex.sh:28: command not found: ^M
./run_bepinex.sh:31: command not found: ^M
./run_bepinex.sh:50: parse error near in^M

```

### BepInEx Distribution

Stable from GitHub

### Environment

```markdown
- OS: Linux 6.1.62-1-MANJARO x86_64 GNU/Linux
- BepInEx: 5.4.22 Linux
- Game: None
```

**Comment by toebeann** [comment]
Duplicate of #687

Fixed in #700

**Comment by siddhartasr10** [comment]
oops sorry, im blind
